components/GestureSpinnerWheel.js

```diff
@@ -12,13 +12,25 @@ import pizzaLoader from '../assets/images/pizzaLoader.gif'
 const API_URL = 'https://whispering-badlands-07525.herokuapp.com/api/getRestaurants'
```

Comment:
it's probably a good idea to check what the network response code is here too... Is it the heroku server erroring? Is it a malformed graphql response and Yelp api is throwing, etc.

Reply:
It occurs on the first spin of the wheel. If I go back home and spin the wheel again then the app works as it should. The problem reoccurs if the app hasn't been used in a while.

Reply:
But do you know why?

Reply:
https://devcenter.heroku.com/articles/free-dyno-hours

Reply:
As a hacky workaround, maybe you can ping the heroku server on app load.. which should warm up the server before the user gets to the carousel screen?